docs/components/Footer.tsx

```diff
@@ -1,23 +1,25 @@
 import { useRouter } from "next/router";
 import Link from "next/link";
-import { useState, ReactNode, ReactElement } from "react";
+import type { ReactNode, ReactElement } from "react";
+import { useState } from "react";
 import cn from "classnames";
 import { ThemeSwitch } from "nextra-theme-docs";
 import VercelLogo from "./logos/Vercel";
-import { useTurboSite, TurboSite } from "./SiteSwitcher";
+import type { TurboSite } from "./SiteSwitcher";
+import { useTurboSite } from "./SiteSwitcher";
```

Comment:
This is also kinda weird. can't you just do
```
import { useTurboSite, type TurboSite } from "./SiteSwitcher"
```

I'm not going to change it, just curious

Reply:
Should be able to, but I think most linters like them separated. I think it might be for the case of something like the following.

```
import { useTurboSite, type { TurboSite, SomeOtherType, SomeThirdType } } from "./SiteSwitcher"
```
is weirder than
```
import type {
  TurboSite,
  SomeOtherType,
  SomeThirdType 
 } from ...
import { useTurboSite } from ...
```
but I'm not 100% guaranteeing this is the case